custom-networks.py

```diff
@@ -0,0 +1,179 @@
+                        help="CSV file encoding, default is %s" % cvconfig.csv_encoding)
+    parser.add_argument("--delimiter", dest="csv_delimiter",
+                        help="CSV file delimiter, default is %s" % cvconfig.csv_delimiter)
+    parser.add_argument("--filename", dest="filename", help="Use this filename, default is custom-networks.csv",
+                        default="custom-networks.csv")
+    # Main Command Parsing
+    command_group = parser.add_mutually_exclusive_group()
+    command_group.add_argument("--import",
+                               help="Create custom networks from a CSV file\n",
+                               action="store_true", default=False, dest="create")
+
+    command_group.add_argument("--export",
+                               help="Export all cusrom networks into a CSV file\n",
+                               action="store_true", default=False, dest="command_export")
+    command_group.add_argument("--delete",
+                               help="Delete all custom networks",
+                               action="store_true", default=False, dest="delete")
+
+    args = parser.parse_args()
+
+    # Handle Cybervision configuration
+    token = set_conf(args.token, cvconfig.token)
+    center_ip = set_conf(args.center_ip, cvconfig.center_ip)
+    center_port = set_conf(args.center_port, cvconfig.center_port)
+    csv_encoding = set_conf(args.csv_encoding, cvconfig.csv_encoding)
+    csv_delimiter = set_conf(args.csv_delimiter, cvconfig.csv_delimiter)
+
+    if not token or not center_ip:
+        print("TOKEN and CENTER_IP are mandatory, check cvconfig.py or us --token/--center-ip")
+
+    if args.token:
+        token = args.token
+    if args.center_ip:
+        center_ip = args.center_ip
+    if args.filename:
+        csv_file = args.filename
+    if not token or not center_ip:
+        print("TOKEN and CENTER_IP are mandatory, check config.py or use --token/--center-ip")
+
+    if args.create:
+        if not csv_file:
+            print("Missing required parameter: --csv-file path_to_csv_file")
+            print_csv_file_format()
+            return
+        create_networks(center_ip=center_ip, center_port=center_port, token=token, csv_file=csv_file,
+                        csv_delimiter=csv_delimiter)
+        return
+    if args.delete:
+        delete_networks(center_ip=center_ip, center_port=center_port, token=token)
+        return
+    if args.command_export:
+        response = get_network_data(center_ip=center_ip, center_port=center_port, token=token)
+        csv_data = convert_json_to_csv(response, csv_delimiter=csv_delimiter)
+
+        with open(csv_file, "w", encoding=csv_encoding) as file:
+            file.write(csv_data)
+        print(csv_data)
+        return
+    parser.print_help()
+
+
+def print_csv_file_format():
+    print("Valid CSV file format example")
+    print("name,type,ip_range,vlan_id,duplicated,split_devices_per_sensor")
```

Comment:
Maybe we can add one additional info 

print("Note: In case name has a comma, then, provide the name in double quotes")

Reply:
Fixed